### PR TITLE
added function detecting quantuminsert MOTS

### PR DESCRIPTION
**Pull request by Aman123lug**
added function for detecting quantuminsert MOTS attacks using findject

### CODE REVIEW

**Comment by AlyaGomaa** [comment]
In your PR, You missed the following points about how slips works
1. i see you used "`print >> sys.stderr`"" but in slips we have an output module, which is like a custom logging library built for slips. so the previous line would look something like this` self.print("hello", 0, 1)`
2. you have a usage msg specifically for the http module, but the usage should be for the main slips.py, because the main slips.py is like the entry point to slips, this is the script we run, we give it the args, then, it starts all the other modules including the HTTP process.
3. you're expecting a pcap list to be passed in argv, but slips only accepts 1 pcap

4. you're using args from argv in a module in the following code:
```
if len(sys.argv) < 2:
    sys.exit('Usage: %s dump.pcap' % sys.argv[0])

```
but we're not 100% sure that argv[0] will be the pcap given to slips, since slips supports many different files othe than pcaps, i usually check for the given type before using it, so here's how i'd do it
```
# make sure my function only works when slips is given a pcap
if __database__.get_input_type() == 'pcap':
    # get the path of the given pcap
    pcap = __database__.get_input_file_path()
```

5. the run() function of every module is like the main() function of it, normally we put the code we want to run in the While True right? but since we want this code to be run once per pcap, we should put the code the calls find_injections() before the While True, so the code would look something like this
```
def run():
    # you're now inside the main function 

    # make sure this function only runs when slips is given a pcap
    if __database__.get_input_type() == 'pcap':
        # get the path of the given pcap
        pcap = __database__.get_input_file_path()
        find_injections(pcap)
    
    while True:
        ....
```


6. no need to use a multiprocessing manager an pool, this is all handled in the main slips.py
7. all dependencies like dpkt and repoze.lru should be added to requirements.txt 
8. it would be awesome if you add a small docstring describing what each function does and use more comments describing what the code does
9. when an injection is found, Slips should alert right? so in the injection_found() function i would add the logic to set an evidence , you can find this in flowalerts module in the file called set_evidence.py

TLDR; The code that does the detection of the MOTS attack is already implemented and the link of it is in the issue description. so the PR should integrate this code in slips, not just copy paste it. but hey thank you so much for the effort! i won't be merging the PR until it's fixed.
